test/computations/main.go

```diff
@@ -53,17 +54,21 @@ func (s *svc) Run(ipAdress string, reqChan chan *manager.ComputationRunReq) {
 	}
 	algoHash := sha3.Sum256(algo)
 	dataHash := sha3.Sum256(data)
-	reqChan <- &manager.ComputationRunReq{
-		Id:              "1",
-		Name:            "sample computation",
-		Description:     "sample descrption",
-		Datasets:        []*manager.Dataset{{Id: "1", Provider: "provider1", Hash: dataHash[:]}},
-		Algorithm:       &manager.Algorithm{Id: "1", Provider: "provider1", Hash: algoHash[:]},
-		ResultConsumers: []string{"consumer1"},
-		AgentConfig: &manager.AgentConfig{
-			Port:        "7002",
-			LogLevel:    "debug",
-			AttestedTls: attestedTLS,
+	reqChan <- &manager.ServerStreamMessage{
+		Message: &manager.ServerStreamMessage_RunReq{
+			RunReq: &manager.ComputationRunReq{
+				Id:              "1",
+				Name:            "sample computation",
```

Comment:
What is this naming (`sample ...`)? Looks like some test, but it is actually in `main()`

Reply:
this is the test server which we use as an example for users as a standin for backends service